cdap-app-templates/cdap-etl/cdap-etl-lib/src/test/java/co/cask/cdap/templates/etl/realtime/sources/JmsMessageToStringSourceTest.java

```diff
@@ -215,18 +199,45 @@ private void verifyEmittedText(JmsSource source) {
     SourceState sourceState = new SourceState();
     source.poll(emitter, sourceState);
 
-    for (String val : emitter.getCurrentValues()) {
-      Assert.assertEquals(originalMessage, val);
+    for (StructuredRecord val : emitter.getCurrentValues()) {
+      String message = val.get(JmsSource.MESSAGE);
+      Assert.assertEquals(originalMessage, message);
 
-      System.out.println("Getting JMS Message in emitter with value: " + val);
+      System.out.println("Getting JMS Message in emitter with value: " + val.get(JmsSource.MESSAGE));
     }
   }
 
+  private void initializeJmsSource(String destination, int messageReceive) {
+    jmsSource = new JmsSource(new JmsSource.JmsConfig(destination, messageReceive));
+
+    jmsSource.configure(new StageConfigurer() {
```

Comment:
Do we need to invoke the configure method anymore? This will be remove once all plugins have been ported.